UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPicker.swift

```diff
@@ -83,11 +83,11 @@ extension UnsplashPhotoPicker: UnsplashPhotoPickerViewControllerDelegate {
     func unsplashPhotoPickerViewController(_ viewController: UnsplashPhotoPickerViewController, didSelectPhotos photos: [UnsplashPhoto]) {
         trackDownloads(for: photos)
         photoPickerDelegate?.unsplashPhotoPicker(self, didSelectPhotos: photos)
-        dismiss(animated: true, completion: nil)
+        //dismiss(animated: true, completion: nil)
```

Comment:
@GweebarraRover Usually we delete the unused code.

@pavlolinearity Should we do it even on this forked repo? Or in this case makes sense leaving the old code commented maybe?